scripts/tasks/sandbox-parts.ts

```diff
@@ -93,7 +93,7 @@ export const install: Task['run'] = async ({ sandboxDir }, { link, dryRun, debug
     // of any storybook packages as verdaccio is not able to both proxy to npm and publish over
     // the top. In theory this could mask issues where different versions cause problems.
     await addPackageResolutions({ cwd, dryRun, debug });
-    await configureYarn2ForVerdaccio({ cwd, dryRun, debug });
+    await configureYarn2ForVerdaccio({ cwd, dryRun, debug, sandboxDir });
```

Comment:
Isn't `cwd === sandboxDir`?

Reply:
Also, I think using `key` here is more robust than relying on a dir name. Key can then also be typed properly

Reply:
you're right, I was copy-pasting too much, `key` is much better.